### ISSUE TITLE
A way to set the canvas size?

### ISSUE THREAD

**Issue by ArtemGr** [SiegeLord/RustGnuplot#21]
Resizing the graph should be as simple as "set term pbm size 600, 400" if I'm reading the [docs](http://gnuplot.sourceforge.net/docs_4.2/node31.html) right. But I wasn't able to find something like that in the library. Is this a missing option?

**Comment by SiegeLord** [comment]
Yeah, this isn't supported yet nicely although you can hack it currently by doing `figure.set_terminal("pbm size 600, 400", "")`. It makes sense to disallow that in the future and correctly expose these arguments via some type.
